NetKAN/TweakScaleRescaled.netkan

```diff
@@ -0,0 +1,32 @@
+spec_version: v1.18
+identifier: TweakScaleRescaled
+name: TweakScale Rescaled
+abstract: >-
+  Rescale parts in the vessel editor.  More features, fewer bugs.
```

Comment:
Might want to omit the abstract so it can be pulled from SpaceDock's short description string and more easily changed without pull request.